modules/standard/IO.chpl

```diff
@@ -6348,6 +6349,35 @@ proc readf(fmt:string):bool {
 }
 
 
+/*
+   Skip a field in the current aggregate format. This method is currently only
+   supported for JSON format and returns ENOTSUP for other formats. In other
+   formats, it may not be possible in general to know when a field ends.
+
+   :arg error: optional argument to capture an error code. If this argument
+               is not provided and an error is encountered, this function
+               will halt with an error message.
+ */
+proc channel.skipField(out error:syserr) {
```

Comment:
Sorry, I misspoke. It does skip the field name part also. But not e.g. a comma afterwards.